app/src/main/AndroidManifest.xml

```diff
@@ -37,6 +37,11 @@
       android:theme="@style/Clinic.V2.Theme.BottomSheetActivity"
```

Comment:
The next commit is `f1a9748b Change Text -> TextInputLayout for labels in CustomPrescriptionEntrySheet`.
Please go to that after this commit is reviewed.